internal/core/workers/operation_execution_worker/operation_execution_worker.go

```diff
@@ -152,7 +152,7 @@ func (w *OperationExecutionWorker) Start(ctx context.Context) error {
 
 		executeStartTime := time.Now()
 		executionErr := executor.Execute(operationContext, op, def)
-		reportOperationExecutionLatency(executeStartTime, w.schedulerName, op.DefinitionName, executionErr != nil)
+		reportOperationExecutionLatency(executeStartTime, w.schedulerName, op.DefinitionName, executionErr == nil)
```

Comment:
I think that is not a good approach in Golang to test if something is equal to nil ([link](https://yourbasic.org/golang/gotcha-why-nil-error-not-equal-nil/)). What do you think to remove this label `isSuccess` in favor of an exclusive metric `errorCount`?

Reply:
Actually, the link you shared warns about the possibility of returning a nil value of a specific error type that can cause problems on checking both `err == nil` and `err != nil`... I think this PR is not the ideal context to make this change and create an abstraction of operation execution error.